custom/icds_reports/views.py

```diff
@@ -1261,13 +1261,13 @@ def get(self, request, *args, **kwargs):
 
         config = {
             'month': tuple(test_date.timetuple())[:3],
-            'aggregation_level': 1,
         }
         location = request.GET.get('location_id', '')
         config.update(get_location_filter(location, self.kwargs['domain']))
         loc_level = get_location_level(config.get('aggregation_level'))
 
         data = []
+        config['aggregation_level'] = 5
```

Comment:
Why this change?

Reply:
@emord Currently in denominator we must sum all records (passing the filters) where ```infra_last_update_date == null```, so we can't take data for specific location level (we will get max 1), we must get records for all awcs in selected location.

Reply:
@emord yes, even if a user is looking for the state level, data would be aggregate based on AWC data.